src/Makefile

```diff
@@ -24,7 +24,7 @@ LIB		=	-lpthread -lpolarssl -L. -L/usr/local/lib
 CHARDEN		=	-fstack-protector -fPIC -D_FORTIFY_SOURCE=2 -O1
 LHARDEN		=	-pie -fPIE
 WFLAGS		=	-Wall -Werror -Wextra
-CFLAGS		=	$(WFLAGS) $(DEFINES) $(INC) $(CHARDEN)
+CFLAGS		+=	$(WFLAGS) $(DEFINES) $(INC) $(CHARDEN)
```

Comment:
The Fedora package spec hands over LDFLAGS to gnu make as a program argument, see [1] e.g. in line 55:
```
...
make fuse %{?_smp_mflags} WFLAGS="$RPM_OPT_FLAGS" LDFLAGS="%{?__global_ldflags}"
...
```
 Doing so causes gnu make to ignore all assignments to LDFLAGS variable within the makefile unless the override keyword is used, see [2]. The same result can be achieved using environment variables and calling make with "-e", see examples 2, 3 and 5:

```
#!/usr/bin/make -f
#
# Example 1:
#
# $ TEST1="1B" TEST2="2B" TEST3="3B" TEST4="4B" TEST5="5B" TEST6="6B" TEST7="7B" TEST8="8B" TEST9="9B" make
#   TEST1 was 1A in Makefile, now it's: 1A
#   TEST2 was 2A in Makefile, now it's: 2B 2A
#   TEST3 was 3A in Makefile, now it's: 3A
#   TEST4 was 4A in Makefile, now it's: 4B 4A
#   TEST5 was not set in Makefile, now it's: 5B
#   TEST6 was 6A in Makefile, now it's: 6A
#   TEST7 was 7A in Makefile, now it's: 7B 7A
#   TEST8 was 8A in Makefile, now it's: 8A 8C
#   TEST9 was 9A in Makefile, now it's: 9B 9A 9C
#
# Example 2:
#
# $ TEST1="1B" TEST2="2B" TEST3="3B" TEST4="4B" TEST5="5B" TEST6="6B" TEST7="7B" TEST8="8B" TEST9="9B" make -e
#   TEST1 was 1A in Makefile, now it's: 1B
#   TEST2 was 2A in Makefile, now it's: 2B
#   TEST3 was 3A in Makefile, now it's: 3A
#   TEST4 was 4A in Makefile, now it's: 4B 4A
#   TEST5 was not set in Makefile, now it's: 5B
#   TEST6 was 6A in Makefile, now it's: 6A
#   TEST7 was 7A in Makefile, now it's: 7B 7A
#   TEST8 was 8A in Makefile, now it's: 8B 8C
#   TEST9 was 9A in Makefile, now it's: 9B 9C
#
# Example 3:
#
# $ make TEST1="1B" TEST2="2B" TEST3="3B" TEST4="4B" TEST5="5B" TEST6="6B" TEST7="7B" TEST8="8B" TEST9="9B"
#   TEST1 was 1A in Makefile, now it's: 1B
#   TEST2 was 2A in Makefile, now it's: 2B
#   TEST3 was 3A in Makefile, now it's: 3A
#   TEST4 was 4A in Makefile, now it's: 4B 4A
#   TEST5 was not set in Makefile, now it's: 5B
#   TEST6 was 6A in Makefile, now it's: 6A
#   TEST7 was 7A in Makefile, now it's: 7B 7A
#   TEST8 was 8A in Makefile, now it's: 8B 8C
#   TEST9 was 9A in Makefile, now it's: 9B 9C
#
# Example 4:
#
# $ export TEST1=1B; export TEST2=2B; export TEST3=3B; export TEST4=4B; export TEST5=5B; export TEST6=6B; export TEST7=7B; export TEST8=8B; export TEST9=9B;
# $ make
#   TEST1 was 1A in Makefile, now it's: 1A
#   TEST2 was 2A in Makefile, now it's: 2B 2A
#   TEST3 was 3A in Makefile, now it's: 3A
#   TEST4 was 4A in Makefile, now it's: 4B 4A
#   TEST5 was not set in Makefile, now it's: 5B
#   TEST6 was 6A in Makefile, now it's: 6A
#   TEST7 was 7A in Makefile, now it's: 7B 7A
#   TEST8 was 8A in Makefile, now it's: 8A 8C
#   TEST9 was 9A in Makefile, now it's: 9B 9A 9C
#
# Example 5:
#
# $ export TEST1=1B; export TEST2=2B; export TEST3=3B; export TEST4=4B; export TEST5=5B; export TEST6=6B; export TEST7=7B; export TEST8=8B; export TEST9=9B;
# $ make -e
#   TEST1 was 1A in Makefile, now it's: 1B
#   TEST2 was 2A in Makefile, now it's: 2B
#   TEST3 was 3A in Makefile, now it's: 3A
#   TEST4 was 4A in Makefile, now it's: 4B 4A
#   TEST5 was not set in Makefile, now it's: 5B
#   TEST6 was 6A in Makefile, now it's: 6A
#   TEST7 was 7A in Makefile, now it's: 7B 7A
#   TEST8 was 8A in Makefile, now it's: 8B 8C
#   TEST9 was 9A in Makefile, now it's: 9B 9C

TEST1=1A
TEST2+=2A
override TEST3=3A
override TEST4+=4A

#TEST5 is not set

override TEST6=6A
TEST6+=6C

override TEST7+=7A
TEST7+=7C

TEST8=8A
override TEST8+=8C

TEST9+=9A
override TEST9+=9C

all:
	@echo "TEST1 was 1A in Makefile, now it's: $(TEST1)"
	@echo "TEST2 was 2A in Makefile, now it's: $(TEST2)"
	@echo "TEST3 was 3A in Makefile, now it's: $(TEST3)"
	@echo "TEST4 was 4A in Makefile, now it's: $(TEST4)"
	@echo "TEST5 was not set in Makefile, now it's: $(TEST5)"
	@echo "TEST6 was 6A in Makefile, now it's: $(TEST6)"
	@echo "TEST7 was 7A in Makefile, now it's: $(TEST7)"
	@echo "TEST8 was 8A in Makefile, now it's: $(TEST8)"
	@echo "TEST9 was 9A in Makefile, now it's: $(TEST9)"
```

Nevertheless i will bring back the override keyword for LDFLAGS and CFLAGS in a upcoming commit. So we don't break fedora's packages...

[1] https://apps.fedoraproject.org/packages/dislocker/sources/spec
[2] http://stackoverflow.com/a/2826178